src/components/LineChart/LineChart.props.ts

```diff
@@ -2,10 +2,12 @@ export interface IChartProps {
     title: string;
     width: number;
     height: number;
-    data: IChart[];
+    gridSize?: number;
+    data: IChartData[];
+    xAxisScale: string; // 'LINEAR' or 'TIME' 
 }
 
-export interface IChart {
-    time: Date;
+export interface IChartData {
+    key: Date | number;
```

Comment:
Personally i would prefer 'argument' and value as names(for all chart types). see: https://documentation.devexpress.com/#CoreLibraries/DevExpressXtraChartsSeriesPointClassPropTopic
yeah, yeah, it's windows forms.... But i believe that the naming is correct.